src/content/en/fundamentals/instant-and-offline/get-started/httprequests-5.md

```diff
@@ -0,0 +1,265 @@
+   display: inline-block;
+   width: 64px; height: 64px;
+   background-image: url("socialmediaicons.png");
+   background-position: 0px 0px;
+   }
+a.twitter {
+   display: inline-block;
+   width: 64px; height: 64px;
+   background-image: url("socialmediaicons.png");
+   background-position: -64px 0px;
+   }
+a.pinterest {
+   display: inline-block;
+   width: 64px; height: 64px;
+   background-image: url("socialmediaicons.png");
+   background-position: -128px 0px;
+   }
+```
+
+Note the extraneous `background-position` property in the `facebook` class. It's not 
+necessary, as the default position is 0,0 but is included here for consistency. The other 
+two classes simply shift the image left horizontally relative to its container by 64px and 
+128px, respectively, leaving the appropriate image section visible in the 64-by-64 pixel 
+link "window".
+
+Here's some HTML to use with that CSS.
+
+```
+<p>Find us on:</p>
+<p><a class="facebook" href="www.facebook.com"></a></p>
+<p><a class="twitter" href="www.twitter.com"></a></p>
+<p><a class="pinterest" href="www.pinterest.com"></a></p>
+```
+
+Instead of including separate images in the links themselves, just apply the CSS classes 
+and leave the link content empty. This simple technique saves you two HTTP requests by 
+letting CSS do the image shifting behind the scenes. If you have lots of small images -- 
+navigation icons or function buttons, for example -- it could save you many, many trips 
+to the server.
+
+You can find a brief but excellent article about this technique, including working examples, 
+at [WellStyled](http://wellstyled.com/css-nopreload-rollovers.html).
+
+## A Caveat
+
+In our discussion of combining text and graphics, we should note that the 
+newer HTTP/2 protocol may change how you consider combining resources. For example, common and 
+valuable techniques like minification, server compression, and image optimization should be 
+continued on HTTP/2. However, physically combining files as discussed above might not achieve 
+the desired result on HTTP/2. 
+
+This is primarily because server requests are faster on HTTP/2, so combining files to 
+eliminate a request may not be substantially productive. Also, if you combine a fairly static 
+resource with a fairly dynamic one to save a request, you may adversely affect your caching 
+effectiveness by forcing the static portion of the resource to be reloaded just to fetch 
+the dynamic portion.
+
+The features and benefits of HTTP/2 are worth exploring in this context. A great place to 
+start is this Google Web Fundamentals article, 
+[Introduction to HTTP/2](https://developers.google.com/web/fundamentals/performance/http2/).
+
+## JavaScript Position and Inline Push
+
+We're assuming so far that all CSS and JavaScript resources exist in external files, and 
+that's generally the best way to deliver them. There are, however, two positional factors 
+about JavaScript that are worth considering.
+
+### Script Location
+
```

Comment:
I think this section would benefit from a read of some of our other script loading guidance. Because of the way modern browsers work, there's some additional nuance to the head/body/position guidance here. A few docs that might help with framing:

https://developers.google.com/speed/docs/insights/BlockingJS
https://www.html5rocks.com/en/tutorials/speed/script-loading/
https://developers.google.com/web/fundamentals/performance/critical-rendering-path/adding-interactivity-with-javascript
https://bitsofco.de/async-vs-defer/